samples/sample_encode/include/pipeline_encode.h

```diff
@@ -168,8 +168,9 @@ struct sInputParams
     mfxU16 IntRefCycleDist;
 
     bool bUncut;
-    bool shouldUseShiftedP010Enc;
-    bool shouldUseShiftedP010VPP;
+    bool shouldUseShifted10BitEnc;
```

Comment:
I don't see these variables to be initialized in constructor...

Reply:
It's hard to see something initialized in constructor if there's no explicit constructor defined. Object of this structure is initialized by zeroes explicitly in main(): sInputParams Params = {};